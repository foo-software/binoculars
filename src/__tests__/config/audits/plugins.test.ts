import auditPlugins from '../../../config/audits/plugins';

describe('plugins audit', () => {
  it('should match snapshot', () => {
    expect(auditPlugins('en')).toMatchSnapshot();
  });
});
