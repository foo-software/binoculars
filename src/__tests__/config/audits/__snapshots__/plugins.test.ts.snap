// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`plugins audit should match snapshot 1`] = `
Object {
  "description": "Search engines can't index plugin content, and many devices restrict plugins or don't support them. [Learn more](https://web.dev/plugins/).",
  "title": "Document avoids plugins",
}
`;
